### PR TITLE
adds 707 texts implements a new menu and makes the proper render of a…

### PR DESCRIPTION
**Pull request by tiagonbotelho**
- Add 707 texts from [this link](http://www.seanwrona.com/typeracer/texts.php?sort=id);
- Remove menu easy/medium/hard option
- Design fixes
- Proper rendering of a random file (was not working before)

### CODE REVIEW

File: lib/need_for_type/display_window.rb [hunk 1]
```diff
@@ -3,46 +3,27 @@
...
     def initialize
       super(Curses.lines, Curses.cols, 0, 0)
       Curses.curs_set(CURSOR_INVISIBLE)
     end
 
-    def render_menu(selected_option)
+    def render_init_game
       self.render_box do
-        self.render_need_for_type_ascii_art(2, 4)
+        self.render_need_for_type_ascii_art(2, (Curses.cols/2)-OFFSET)
```
Comment (jbernardo95) [inline lib/need_for_type/display_window.rb]: Spacing: `(Curses.cols / 2) - OFFSET`

File: lib/need_for_type/file_manager.rb [hunk 2]
```diff
@@ -2,30 +2,16 @@ module NeedForType
...
-      @path = "#{NeedForType::GEM_ROOT}/assets/#{difficulty_to_s(difficulty)}/"
+    def initialize()
+      @path = "#{NeedForType::GEM_ROOT}/assets/texts/"
     end
 
     # Reads random file from difficulty folder
     def get_random_text
-      directory = Dir[@path]
-      random_file = rand(1..directory.count)
+      directory_count = Dir[File.join(@path, '**', '*')].count { |file| File.file?(file) }
```
Comment (jbernardo95) [inline lib/need_for_type/file_manager.rb]: Why this change ?

`.count { |file| File.file?(file) }` this portion is kind of confusing, so much `file` 🙇

File: lib/need_for_type/states/start.rb [hunk 3]
```diff
@@ -10,9 +10,11 @@ def initialize(display_window)
     end
 
     def update
-      @display_window.render_menu(@option)
+      @display_window.render_init_game
```
Comment (jbernardo95) [inline lib/need_for_type/states/start.rb]: I think we should rename `render_init_game` function to `render_start`, to make it more clear.

And following the same pattern we should also rename `render_score` to `render_end`, so that every state has it's own rendering function.

File: lib/need_for_type/states/game.rb [hunk 4]
```diff
@@ -54,15 +54,9 @@ def handle_init_game
     end
 
     def handle_start_game
```
Comment (jbernardo95) [inline lib/need_for_type/states/game.rb]: I think we can merge the `init_game` sub state into the `start_game` sub state because there is no more need to have an `init_game_state`.

**Review by jbernardo95 — CHANGES_REQUESTED** [review]
@tiagonbotelho Left some comments, please check them out.

Other things:

Missing things:
- Update `CHANGELOG.md` accordingly
- Update `README.md` screenshot for the updated welcome screen
- Update `README.md` credits section to give credit to the texts source

Suggestions:
- Render the ascii art logo in green

Bug:
- There is a rendering bug when I ran `need_for_type` on a 80x25 terminal window (check images on a following comment)

**Comment by jbernardo95** [comment]
@tiagonbotelho Here is the rendering bug:

screen shot 2016-09-24 at 20 13 46
screen shot 2016-09-24 at 20 13 50

It is impossible to support all terminal sizes, but I think we should at least support small windows (80x25) and bug windows (160x50).

File: lib/need_for_type/file_manager.rb (hunk 2 above)
Comment (tiagonbotelho) [reply lib/need_for_type/file_manager.rb]: it was used to verify if the file was actually a file or not but this is not needed for now so I will leave it at the count ;)

**Review by jbernardo95 — APPROVED** [review]

**Comment by jbernardo95** [comment]
@tiagonbotelho Awesome !

Updated the screen shot and merged.
